### ISSUE TITLE
Discrepancy in Model Performance on water-tank/brick6 Dataset

### ISSUE THREAD

**Issue by TanngJak** [sollynoay/A2FNet#4]
<img width="2360" height="952" alt="Image" src="https://github.com/user-attachments/assets/c423dec5-29e2-4eae-ad46-864815c6b950" />

Thank you for your work.

I'm attempting to reproduce the reported performance on the water-tank/cylinder10 dataset but am encountering a slight discrepancy, particularly regarding the MAE with L1 loss and the visual quality of the output.

Training Epochs: 700 epochs
Model Selection for Evaluation: The model with the lowest validation loss was selected for evaluation.
Average Validation Loss of Selected Model: 0.1509
MAE with L1 Loss: My trained model achieved an MAE of 0.3308. This MAE value is slightly higher than the one reported in the paper.
Visual Quality: I've observed that the generated edges in the output are somewhat ambiguous, which might contribute to the higher MAE.

I would greatly appreciate any advice you could offer regarding the somewhat ambiguous edges and the quality of the pseudo front depth maps in my results. Could you please explain potential reasons why they might not be matching the quality shown in your work?

![Image](https://github.com/user-attachments/assets/89614755-dc48-4a2b-b043-0693d14c7aa4)
![Image](https://github.com/user-attachments/assets/c690d9cc-f396-4e15-bdff-909478c04b4d)
